map/climate.txt

```diff
@@ -1,34 +1,106 @@
+severe_winter = {
+	# DRAKKARI / Drak'tharon
+	766 767 768 769 770 771 772 773 774 775 
+	# DRAKKARI / Frostmane
+	142 143 151 152 153 944 945 946 947 
+	# DRAKKARI / Zul'drak
+	754 755 756 757 758 759 760 761 762 763 764 765 
+	# Icecrown
+	695 
+	# Isle of Conquest
+	938 939 940 941 942 
+	# KALDOREI / Hyjal
+	1921 1922 1923 1924 1925 1926 1927 1928 1929 1930 1931 1932 1933 1934 1935 1936 1937 1938 1939 1940 1941 1942 1943 1944 1945 1946 1967 1968 1969 1970 1971 
+	# KALDOREI / Winterspring
+	1947 1948 1949 1950 1951 1952 1953 1954 1955 1956 1957 1958 1959 1960 1961 1962 1963 1964 1965 1966 
+	# KHAZ MODAN / Gnomeregan
+	146 147 148 149 150 948 950 
+	# KHAZ MODAN / Ironforge
+	102 132 133 134 135 136 137 138 139 140 141 144 145 154 155 156 157 158 159 160 161 162 163 164 165 
+	# NO KINGDOM
+	2001 2002 2003 2004 2005 2006 2007 2008 2009 
+	# STORM PEAKS / Crystalsong Forest
+	927 928 929 930 
+	# STORM PEAKS / Dun Niffelem
+	866 867 868 881 882 883 884 885 886 887 
+	# STORM PEAKS / Icecrown
+	907 908 909 910 911 912 913 914 915 916 917 918 919 920 921 922 923 924 925 926 
+	# STORM PEAKS / Storm Peaks
+	872 873 874 875 876 877 878 879 880 
+	# STORM PEAKS / Ulduar
+	858 864 869 870 871 1144 
+	# TAUNKA WASTELANDS / Borean Tundra
+	818 819 820 821 822 823 824 825 826 827 828 830 832 833 834 835 836 837 838 839 840 841 842 843 844 845 846 847 848 849 
+	# TAUNKA WASTELANDS / Dragonblight
+	777 778 779 780 781 782 783 784 785 786 787 788 789 790 791 792 793 794 795 796 797 798 799 800 801 802 803 804 805 806 817 1161 1162 1163 1164 1165 
```

Comment:
817 is Strand of Ancients, which has a jungle-like biome, therefore the winter shouldn't be present there

Reply:
Fixed in d652cfd342637425ada90a0be82b68acf32a5c98.